src/strkey.js

```diff
@@ -64,11 +65,11 @@ export class StrKey {
 
   /**
    * Decodes strkey ed25519 seed to raw data.
-   * @param {string} data data to decode
+   * @param {string} address data to decode
    * @returns {Buffer}
    */
-  static decodeEd25519SecretSeed(data) {
-    return decodeCheck('ed25519SecretSeed', data);
+  static decodeEd25519SecretSeed(address) {
+    return decodeCheck('ed25519SecretSeed', address);
```

Comment:
💡 Fwiw the secret seed strkey is not an address, it's a seed encoded in a strkey. Using the term `encoded` instead of `data` may be more accurate if you're wanting to get away from the generic `data`.

Reply:
Yeah that's a pretty good point... I wanted to get away from the generic `data` to indicate the _type_. Calling it `encoded` doesn't help, since that's implied by `decodeEd25...`. I find myself checking the docstring every single time I use this set of helpers, because "am I encoding to strkey or am I decoding from strkey or am I decoding from binary or am I ..." etc.

Reply:
Naming it `str` may help then.